rules/linux/privilege_escalation_apt_binary.toml

```
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary apt/apt-get abuse to breakout out of restricted shells or environments by spawning an
+interactive system shell.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via  apt /apt-get Changelog Escape"
+references = [
+    "https://gtfobins.github.io/gtfobins/apt/",
+    "https://gtfobins.github.io/gtfobins/apt-get/"
+]
+risk_score = 47
+rule_id = "8fed8450-847e-43bd-874c-3bbf0cd425f3"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation","GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by process.pid with maxspan=1m
```

To avoid FP from pid reuse / host collisions

```suggestion
sequence by process.entity_id with maxspan=1m
```